File: Lecture024 Maths for DSA/primeNumberBrute.cpp
#include<iostream>
using namespace std;

bool isPrime(int n) {

    if(n<=1)
    return false;

    for(int i=2; i<n; i++)
    {
        if(n%i==0)
        {
            return false;
        }
    }
    return true;
}

int main() {

    int n;
    cin >> n;

    if(isPrime(n)) {
        cout << "It is a Prime Number" << endl;
    }
    else{
        cout << " It is not a Prime Number" << endl;
    }


    return 0;
}

File: Lecture024 Maths for DSA/primeNumberBrute.py

def isPrime( n ):

    if(n<=1):
        return False

    for i in range(2,n):
    
        if(n%i==0):
            return False
    return True

if __name__=="__main__":

    n = int(input())

    if(isPrime(n)):
        print("It is a Prime Number")
    
    else:
        print(" It is not a Prime Number")
    